### ISSUE TITLE
[BUG] Page Listing titles inconsistent use of <strong>

### ISSUE THREAD

**Issue by jporich** [unimelb/design-system-issues#53]
At present Page Listings make use of the metadata **keyword** field to insert a 'tag'/'category' before the title.

- When the keyword field is used, the title is bolded with a `<strong>` tag
- When the keyword field is _not_ used, the title doesn't receive a `<strong>` tag

## Expected behaviour
Title should always receive a bold tag, regardless of whether the keyword metadata field is being used or not.

## Current behaviour
![Screen Shot 2020-02-03 at 10 40 22 am](https://user-images.githubusercontent.com/10070570/73617426-a663dd00-4672-11ea-924a-2d5fe4b39ef7.png)

At present the titles with no keyword applied match the font-weight of the categories (i.e. assets making use of the 'keyword' field). This creates a sense of visual consistency between the titles and the keyword tags - which shouldn't be the case

## Steps to reproduce
https://cms.unimelb.edu.au/controlled-environment/law/current/study-options/_nocache

Create a page listing - give some bottom-tier assets a keyword, remove from others

### Affected browsers
Chrome, FF

## Possible solution
Always give the title a `<strong>` tag - if keyword is used, insert it before the title without a `<strong>` tag.

**Comment by jporich** [comment]
NOTE: Here is a workaround with `&nbsp;` in the metadata keyword field
![Screen Shot 2020-02-03 at 11 05 50 am](https://user-images.githubusercontent.com/10070570/73617750-23448600-4676-11ea-87e8-a6fb4c3d6e37.png)


--------

Ideally the keyword would be pinned to the bottom (filling 'upwards'). This allows for better alignment if the keyword isn't present, and has the added bonus of giving a better visual hierarchy that the keyword is _second_ to the title.

Would introduce complications for long titles/keywords, however it seems like this would be a rare edge case.

See photoshopped version below: 

![Screen Shot 2020-02-03 at 11 07 30 am](https://user-images.githubusercontent.com/10070570/73617777-5d158c80-4676-11ea-9058-8cdb33e27343.png)

**Comment by tomstringer** [comment]
@dkevey Is this a Matrix-specific implementation issue? If so, I think we should shift this across to unimelb/matrix-controlled-environment

**Comment by dkevey** [comment]
Yes @tomstringer - would be good to convert this all to a content template too I think so we can add extra config options
